docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -0,0 +1,366 @@
+"""
+.. _lightly-custom-augmentation-5:
+
+Tutorial 5: Custom Augmentations
+==============================================
+
+In this tutorial, we will train a model on chest X-ray images in a self-supervised manner.
+In self-supervised learning, X-ray images can pose some problems: They are often more 
+than eight bits deep which makes them incompatible with certain standard torchvision 
+transforms such as, for example, random-resized cropping. Additionally, some augmentations
+which are often used in self-supervised learning are ineffective on X-ray images.
+For example, applying color jitter to an X-ray image with a single color channel 
+does not make sense.
+
+We will show how to address these problems and how to train a ResNet-18 with MoCo
+on a set of 16-bit X-ray images in TIFF format.
+
+The original dataset this tutorial is based on can be found `on Kaggle <https://www.kaggle.com/c/vinbigdata-chest-xray-abnormalities-detection/overview>`_.
+These images are in the DICOM format. For simplicity and efficiency reasons, 
+we randomly selected ~4000 images from the above dataset, resized them such that the
+maximum of the width and height of each image is no larger than 512, and converted
+them to the 16-bit TIFF format. To do so, we used ImageMagick:
```

Comment:
Maybe add a hint on how to install or that it's already installed on most Linux systems.